lib/composer.js

```diff
@@ -318,13 +322,12 @@ export default class Composer extends Disposable {
     latex.status.setIdle()
   }
 
-  async build (shouldRebuild) {
+  async build (shouldRebuild = false, silent = false) {
```

Comment:
I think `quiet` is more frequently used, but what about making this more verbose and understandable; `suppressLogging`. Alternatively, flip the logic and name it `verbose` or some such.

Reply:
What do think about `enableLogging`?

Reply:
I like it :1st_place_medal: